pkg/solana/chainwriter/helpers.go

```diff
@@ -41,12 +42,15 @@ func GetValuesAtLocation(args any, location string) ([][]byte, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	for _, value := range addressList {
 		if byteArray, ok := value.([]byte); ok {
 			vals = append(vals, byteArray)
 		} else if address, ok := value.(solana.PublicKey); ok {
 			vals = append(vals, address.Bytes())
+		} else if num, ok := value.(uint64); ok {
+			buf := make([]byte, 8)
+			binary.LittleEndian.PutUint64(buf, num)
```

Comment:
btw, out of curiosity, why are we using Little Endian here?
I saw some onchain code in solana used big endian while encoding data.

Reply:
Borsh encoding uses little endian, not sure where you saw the Big Endian usage?